tcms_django_plugin/result.py

```diff
@@ -47,27 +36,27 @@ def addSuccess(self, test):
     def addError(self, test, err):
         super().addError(test, err)
         self.status_id = self.backend.get_status_id('FAILED')
-        self.trace_back = self.errors[-1][1]
+        self.comment = self.errors[-1][1]
 
     def addFailure(self, test, err):
         super().addFailure(test, err)
         self.status_id = self.backend.get_status_id('FAILED')
-        self.trace_back = self.failures[-1][1]
+        self.comment = self.failures[-1][1]
 
     def addSkip(self, test, reason):
         super().addSkip(test, reason)
         self.status_id = self.backend.get_status_id('WAIVED')
-        self.skip_reason = 'Reason test was skipped: {0!r}'.format(reason)
+        self.comment = 'Reason test was skipped: {0!r}'.format(reason)
 
     def addExpectedFailure(self, test, err):
         super().addExpectedFailure(test, err)
         self.status_id = self.backend.get_status_id('FAILED')
-        self.expected_failure_comment = 'Test failed as expected'
-        self.trace_back = self.expectedFailures[-1][1]
+        self.comment = "Test failed as expected.\n\n{0}".format(
+            self.expectedFailures[-1][1])
```

Comment:
Remove the surrounding text, just use the expectedFailure message as it comes from the test framework.